reference/cd-github.md

```diff
@@ -0,0 +1,92 @@
+---
+title: GitHub Workflow (Preview)
+---
+
+<style>
+    .img-bordered {
+        border: 2px solid rgba(0,0,0,0.15);
+    }
+    .move-right {
+        margin-left: 32px;
+    }
+    .move-right-small {
+        margin-left: 8px;
+    }
+</style>
+
+Pulumi's GitHub workflow integration is a feature which shows the results of Pulumi stack
+updates on GitHub. Showing you any potential infrastructure changes on Pull Requests,
+as well as recording stack changes made with each push.
+
+> GitHub workflow integration is currently in preview, and we will be expanding the feature set
+> over the comming weeks and months. If you have any feedback, suggestions, or bug reports, please
+> let us know in the [Pulumi Community Slack](https://slack.pulumi.io/).
+
+
+**Pull Request Comment**
+
+![Comment on Pull Request](/images/github-app/pr-comment.png){:class="img-bordered"}{:class="move-right"}
+
+**Check Suite Result**
+
+![Results on GitHub Check](/images/github-app/checks-detail.png){:width="600px"}{:class="img-bordered"}{:class="move-right"}
```

Comment:
Also could use text. For instance,

The PR comments feature makes it easy to see quick summaries of changes that a PR will make, while the GitHub App's integration with the new [Checks API](https://blog.github.com/2018-05-07-introducing-checks-api/) delivers more complete information about changes, down to the individual resources and their properties.